### PR TITLE
feat: create view wireframe established

### LINKED ISSUE #1: Bootstrap frontend code in `client` folder
Right now in `client` folder, only an index.html is there for testing. As our plan is to use react.js to develop our frontend, it is common practice to use [create-react-app](https://github.com/facebook/create-react-app) to bootstrap your project first.

If it is done correctly. a `package.json` file should be inside `client` folder (ex. client/package.json)

### LINKED ISSUE #3: Develop book review creation view
Wireframe:
codeday-project-wireframe-creation

Feel free to make it more beautiful. :stuck_out_tongue: 

If you don't want to write all the css by yourself, one way is to use component libraries like [material-ui](https://material-ui.com/), [bootstrap](https://react-bootstrap.github.io/) or [semantic](https://react.semantic-ui.com/).

### PR DESCRIPTION
**Pull request by dknv**
using create-react-app for client folder (issue #1) and establishing the layout/design for users' create view (issue #3)

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "templates",
```
Comment (jeromewu) [inline client/package.json]: The name of the package should be `bookworms`.

> If you update the name manually, don't forget to update `package-lock.json` as well.

File: client/src/App.test.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```
Comment (jeromewu) [inline client/src/App.test.js:7]: This test case failed as it expects a link element with text `learn react` inside, it should be refactored to meet current page.

**Review by jeromewu — CHANGES_REQUESTED** [review]
In `client/public` folder, there is a `logo.png` file which is 300+ KB at the moment, this big image file may cause the website loading slow when the network is not fast enough.

It is recommended to convert logo.png to 3 files:
- `client/public/favicon.ico`: image in ico format
- `client/public/logo192.png`: 192x192 image
- `client/public/logo512.png`: 512x512 image

You can use online tools to do the conversion and more details are addressed in `client/public/manifest.json`

Also it is better to resize `client/src/logo.png` if we don't need it to be this high resolution.

BTW, nice logo, I am going to use it as our repo image. :smile:

File: client/src/Home.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+/**
+ * Homepage
+ * 
+ */
+
+function Home() {
+
+    return (
+        <div className = "home">
```
Comment (jeromewu) [inline client/src/Home.js]: Most of the time, people don't keep white spaces before and after `=`, so it is better to write it as:

```jsx
<div className="home">
```

File: client/src/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <nav className = "navbar">
+            <img src = {logo} height={45} width={45} alt="logo"></img>
+            <h1> bookworms </h1>
+            <div className = "links">
+                <Link to ="/">Home</Link>
+                <Link to ="/create" style = {{
+                    color: 'white',
+                    backgroundColor: '#dd9f33',
+                    borderRadius: '8px',
+                }}>Add Review</Link>
```
Comment (jeromewu) [inline client/src/Navbar.js]: Better to use `className` instead of `style` if applicable.

**Review by jeromewu — CHANGES_REQUESTED** [review]

**Review by dknv — COMMENTED** [review]
the requested changes were implemented/fixed in the newer version

File: client/src/Home.js (hunk 3 above)
Comment (jeromewu) [reply client/src/Home.js]: @auroravo2001 I still see same issue in `client/src/NavBar.js`, better to double check again.

**Comment by jeromewu** [comment]
Only one issue left, others are fixed alr. Good job. :+1:

File: client/src/Home.js (hunk 3 above)
Comment (dknv) [reply client/src/Home.js]: @jeromewu so sorry, it took me some time to figure out where it was...!

Comment (jeromewu) [reply client/src/Home.js]: No worries, I know it is hard to see them. Haha

**Review by jeromewu — APPROVED** [review]

**Comment by jeromewu** [comment]
@auroravo2001 OK, right now it is merged! Thanks for the hard work. :smile:

**Pull request merged**